test/namespaces/controllers.spec.js

```diff
@@ -23,9 +23,10 @@ describe('=> NamespacesCtrl tests', function () {
         toastr,
         httpGetNamespaces,
         modalInstance,
-        $translate;
+        $translate,
+        $licenseService;
 
-    beforeEach(angular.mock.inject(function (_$httpBackend_, _$repositories_, _RDF4JRepositoriesRestService_, _toastr_, _$location_, _$controller_, _$window_, _$timeout_, $rootScope, $q, _$translate_) {
+    beforeEach(angular.mock.inject(function (_$httpBackend_, _$repositories_, _RDF4JRepositoriesRestService_, _toastr_, _$location_, _$controller_, _$window_, _$timeout_, $rootScope, $q, _$translate_, _$licenseService_) {
```

Comment:
The unit tests are not run on CI anymore and will be removed

Reply:
Reverted